src/models/property-editor.ts

```diff
@@ -0,0 +1,137 @@
+import * as vlSchema from 'vega-lite/build/vega-lite-schema.json';
```

Comment:
Where should the custom schemas be located? Wasn't sure where to put the custom schemas so put into `/models` for now